monero/test_support.go

```diff
@@ -60,10 +60,8 @@ func CreateWalletClient(t *testing.T) WalletClient {
 }
 
 // GetBalance is a convenience method for tests that assumes you want the primary
-// address, that you want to refresh, and that errors should fail the test.
+// address and that errors should fail the test.
 func GetBalance(t *testing.T, wc WalletClient) *wallet.GetBalanceResponse {
-	err := wc.Refresh()
-	require.NoError(t, err)
```

Comment:
`GetBalance` below now does the refresh. I removed `Refresh` from the outside-package wallet API. If someone calls `GetBalance` or `GetHeight`, the refresh is taken care of for them.